lib/drivers/insulet/insuletDriver.js

```diff
@@ -1788,66 +1757,22 @@ module.exports = (config) => {
           }
 
           // no mounted drive with .ibf file, let's try connecting over MTP (Dash)
-          const tempPath = (electron.app || electron.remote.app).getPath('temp');
-
-          if (process.platform !== 'win32') {
-            const mtp = require('node-mtp');
-            mtp.attach();
-            const list = mtp.getFileListing();
-            debug('Files:', list);
-            const tempFile = path.join(tempPath, 'temp.ibf');
-            const filtered = _.filter(list, (obj) => _.endsWith(obj.filename, '.ibf'));
-            const fileObj = _.maxBy(filtered, 'id');
-            debug('Retrieving', fileObj);
-            mtp.getFile(fileObj.id, tempFile);
-
-            const res = await fs.readFile(tempFile);
-            buf = res.buffer;
-            bytes = new Uint8Array(buf);
-            mtp.release();
-            await fs.unlink(tempFile);
+          const Mtp = require('webmtp');
+          const mtp = new Mtp(deviceInfo.vendorId, deviceInfo.productId);
```

Comment:
```suggestion
          const webmtp = require('webmtp');
          const mtp = new webmtp(deviceInfo.vendorId, deviceInfo.productId);
```

Perhaps a little pedantic, but I always get a bit nervous around variables that only vary by case. It's not bad in this case, because it's only 2 lines. Optional suggestion 😄

Reply:
Added it, and then changed it to `WebMTP`, as our linting rules require constructors to start with a capital letter.